Sources/SwiftParser/Parser.swift

```diff
@@ -46,7 +46,7 @@
 ///     /*  */
 ///     let rbrace = self.expect(.rightBrace)
 ///
-/// Unlike ``TokenConsumer/eat(_:)``, ``Parser/expect(_:remapping:)`` returns
+/// Unlike `TokenConsumer.eat(_:)`, `Parser.expect(_:remapping:)` returns
```

Comment:
Is there no way to link to these (and all the others that had similar treatment)?

Reply:
No, because `TokenConsumer.eat` is not public, it doesn’t show up in the generated documentation and hence there’s no way to link to it.

Reply:
Ah I see 👍